server/db/models/user.js

```diff
@@ -3,10 +3,53 @@ const Sequelize = require('sequelize')
 const db = require('../db')
 
 const User = db.define('user', {
+  firstName: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  lastName: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
   email: {
     type: Sequelize.STRING,
     unique: true,
-    allowNull: false
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+      isEmail: true
+    }
+  },
+  phone: {
+    type: Sequelize.INTEGER,
+    unique: true,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+      isNumeric: true,
+      max: 10,
+      min: 10
+    }
+  },
+  defaultShipping: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  defaultBilling: {
```

Comment:
We should consider if we want to allow null and empty on this.
What if address instance doesn't exist at time of creating user?